_posts/0000-01-02-TRAETHEGREAT.md/my-slide

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!